requirements.txt

```diff
@@ -24,7 +24,4 @@ Werkzeug==0.16.1
 flask
 flask-restplus
 flask-cors
-# optional: for inference with fasttext
```

Comment:
Maybe we should keep the optional ones in here as comments :thinking: 
I think it doesn't hurt, but will provide some helpful information for people who want to use fasttext / onnx. Not everybody will check setup.py